prov/efa/src/rxr/rxr.h

```diff
@@ -169,6 +169,12 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
  */
 #define RXR_RECEIPT_RECEIVED BIT_ULL(7)
 
+/*
+ * Flag to tell that
+ * long message protocol is used
+ */
+#define RXR_LONG_MSG BIT_ULL(8)
```

Comment:
LONG_MSG is wrong because write also use this. Maybe RXR_LONGCTS_PROTOCOL?

Reply:
Flag name changed, resovling.